metashare/api/models.py

```diff
@@ -90,3 +95,14 @@ def valid_token_for(self):
         if all(self.token) and self.org_id:
             return self.org_id
         return None
+
+
+class Product(HashIdMixin):
+    name = models.CharField(max_length=50, unique=True)
+    repo_name = models.SlugField(unique=True)
```

Comment:
Is this just the name, or also the org/owner? Or is it the repo URL?

Reply:
It's the repo _name_ on GitHub, which given the talk about having everything be under one org, can be used to get the full URL.

Reply:
Makes sense. I assume we'll want that org setting controlled by an env var or something, not hard-coded.

Reply:
It's likely that we will only want to configure one GitHub organization, but I don't think we want to build that assumption into the data model. I think we want to store both org and repo name and have a URL structure like /products/[org]/[repo]